client/navigation/components/container/index.js

```diff
@@ -87,6 +87,26 @@ const Container = ( { menuItems } ) => {
 
 	const navDomRef = useRef( null );
 
+	const renderNavigationGroup = ( category, items, topLevelTitle = null ) => {
+		if ( ! items.length ) {
+			return null;
+		}
+
+		const isTopLevel = category.id === 'woocommerce';
+
+		if ( ! isTopLevel && items[ 0 ].menuId !== category.menuId ) {
```

Comment:
Looking at the code, I think we can expect that this would never be called where `items[ 0 ]` had the same menu ID, but `items[ 1 ]` did not?

Reply:
Right, since they're already categorized all items should have the same menu ID, though I think there's probably a better way to handle this.